ext/standard/basic_functions_arginfo.h

```diff
@@ -1603,6 +1603,12 @@ ZEND_BEGIN_ARG_WITH_RETURN_TYPE_MASK_EX(arginfo_abs, 0, 1, MAY_BE_LONG|MAY_BE_DO
 	ZEND_ARG_TYPE_MASK(0, num, MAY_BE_LONG|MAY_BE_DOUBLE, NULL)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_MASK_EX(arginfo_clamp, 0, 3, MAY_BE_LONG|MAY_BE_DOUBLE)
```

Comment:
This is a generated file, you can run `php build/gen_stub.php` to regenerate it (as apparently the build system didn't do this for you).

Reply:
Strange, I've done that generation before, does it need to be regenerated between builds and tests?

Reply:
And now it has a merge conflict. 🙄

Reply:
Should I revert 388b2b5 and do `php build/gen_stub.php` after re-merging in `master`? Might solve the conflict.

Reply:
Yeah, just merging/rebasing master and regenerating should do it.

Reply:
Merging master caused some issues with linking when re-configuring the build process. 🤷‍♂️

```bash
make: *** No rule to make target `sapi/phpdbg/phpdbg_eol.h', needed by `sapi/phpdbg/phpdbg.lo'
```

Any idea on how to fix this issue?

Reply:
Probably needs a `make clean && ./buildconf && ./config.nice && make -j32`. Changes to the build system still require manual invalidation.